src/tests/ut/room/room_test.cpp

```diff
@@ -0,0 +1 @@
+#include "room_test.hpp"
```

Comment:
Why is it empty? 
If no tests will be introduced then remove those files 
room_test.cpp
room_test.hpp

Reply:
Fixed